tests/integration/cmd_push_test.go

```diff
@@ -257,8 +262,6 @@ var _ = Describe("odo push command tests", func() {
 
 			Expect(modifiedCatViewFile).NotTo(Equal(earlierCatViewFile))
 			Expect(modifiedCatServerFile).To(Equal(earlierCatServerFile))
-
-			helper.HttpWaitFor("http://"+url, "UPDATED!", 30, 1)
```

Comment:
Not an issue with moving it just after pushing the component for second time

/lgtm